### ISSUE TITLE
Segmenting Lists - Undefined Property

### ISSUE THREAD

**Issue by gustavo-2830** [spatie/mailcoach-support#95]
Hello,

If a campaign is created using a custom segment, the following error appears when trying to access the settings through the front-end:

```
Facade\Ignition\Exceptions\ViewException
Undefined property: Facade\Ignition\Views\Engines\CompilerEngine::$segment_class (View: mailcoach/resources/views/app/campaigns/draft/partials/emailListFields.blade.php)
```

Also, the documentation regarding how to create a campaign using segments appears to be outdated:

`->html()` and `->useSegment()` are Undefined.

https://mailcoach.app/docs/v1/package/advanced-usage/segmenting-lists

**Comment by freekmurze** [comment]
I've updated docs. 

Hunting down the bug now...

**Comment by freekmurze** [comment]
This has been fixed in `v1.8.1` and `v2.0.1`

Thanks for reporting this.
